packages/web-react/src/components/Collapse/useCollapseAriaProps.ts

```diff
@@ -49,7 +49,7 @@ export const useCollapseAriaProps = (props: CollapseAriaPropsProps): CollapseAri
   const triggerProps = {
     [NAME_ARIA_EXPANDED]: String(isCollapsed) as Booleanish,
     [NAME_ARIA_CONTROLS]: String(id),
-    onClick: toggleHandler,
+    onClick: toggleHandler as (event: ClickEvent) => void,
```

Comment:
Is there some problem with the type that you must enforce the type for this handler?

Reply:
Ha, I test it and see the problem with mismatched type of aria-expanded, which is one time `string` and other time `Booleanish`.

```git
diff --git packages/web-react/src/components/Collapse/__tests__/useCollapseAriaProps.test.ts packages/web-react/src/components/Collapse/__tests__/useCollapseAriaProps.test.ts
index bc1fe40f..fbc153db 100644
--- packages/web-react/src/components/Collapse/__tests__/useCollapseAriaProps.test.ts
+++ packages/web-react/src/components/Collapse/__tests__/useCollapseAriaProps.test.ts
@@ -6,6 +6,7 @@ describe('useCollapseAriaProps', () => {
     const props = {
       id: 'test-collapse-id',
       isCollapsed: true,
+      toggleHandler: () => null,
     };
     const { result } = renderHook(() => useCollapseAriaProps(props));
 
diff --git packages/web-react/src/components/Collapse/useCollapseAriaProps.ts packages/web-react/src/components/Collapse/useCollapseAriaProps.ts
index 8f6c10ac..bb9248ed 100644
--- packages/web-react/src/components/Collapse/useCollapseAriaProps.ts
+++ packages/web-react/src/components/Collapse/useCollapseAriaProps.ts
@@ -10,7 +10,7 @@ export interface CollapseAriaPropsProps {
   id: string;
   isCollapsed: boolean;
   collapsibleToBreakpoint?: CollapseResponsiveType;
-  toggleHandler?: (event: ClickEvent) => void;
+  toggleHandler: (event: ClickEvent) => void;
 }
 
 export interface CollapseAriaPropsReturn {
@@ -47,9 +47,9 @@ export const useCollapseAriaProps = (props: CollapseAriaPropsProps): CollapseAri
     ref: contentElementRef,
   };
   const triggerProps = {
-    [NAME_ARIA_EXPANDED]: String(isCollapsed) as Booleanish,
+    [NAME_ARIA_EXPANDED]: isCollapsed,
     [NAME_ARIA_CONTROLS]: String(id),
-    onClick: toggleHandler as (event: ClickEvent) => void,
+    onClick: toggleHandler,
   };
 
   return {
(END)
```